pkg/client/rootd/dns/server_linux.go

```diff
@@ -105,8 +105,12 @@ func (s *Server) runOverridingServer(c context.Context, dev *vif.Device) error {
 		for _, line := range strings.Split(string(dat), "\n") {
 			if strings.HasPrefix(strings.TrimSpace(line), "nameserver") {
 				fields := strings.Fields(line)
-				s.config.LocalIp = net.ParseIP(fields[1])
-				dlog.Infof(c, "Automatically set -dns=%s", net.IP(s.config.LocalIp))
+				ip := net.ParseIP(fields[1])
+				if ip.To4() != nil {
+					s.config.LocalIp = ip.To4()
+					dlog.Infof(c, "Automatically set -dns=%s", net.IP(s.config.LocalIp))
+					break
```

Comment:
This break is wrong. It will prevent parsing of "search" if that comes after "nameserver".

Reply:
ah ok good catch thank you

Reply:
> What would it take to actually support IPv6?

not sure really i think there might be a way to make iptables work with it (i believe it's a matter of using iptables6 but i've never done it), but rn it fails

Reply:
Not for this release, for sure.